keyboards/h/amharic/index.php

```diff
@@ -6,6 +6,7 @@
   // Required
   head([
     'title' =>'Keyman for Amharic | አማርኛ ይጻፉ',
+    'description' => 'Keyman keyboards for Amharic and other Ethiopic languages',
```

Comment:
Let's SEO these a bit? Not just repeating the content in the page?

Free and open source Amharic and Ethiopic keyboard layouts for Windows, macOS, Linux, Android, iOS and web

Or something like that?